crates/store/re_video/src/demux/mod.rs

```diff
@@ -174,6 +175,9 @@ impl GroupOfPictures {
 /// A single sample in a video.
 #[derive(Debug, Clone)]
 pub struct Sample {
+    /// The start of a new [`GroupOfPictures`]?
```

Comment:
why the question mark? That is how we defined them, no?

Reply:
This is the question the boolean answers